package.json

```diff
@@ -11,6 +11,9 @@
     "build": "lerna run build",
     "check-version-conflicts:beta": "yarn ts-node ./scripts/check-version.ts beta",
     "check-version-conflicts:release": "yarn ts-node ./scripts/check-version.ts",
+    "cloud-e2e-cb": "bash -c 'source ./scripts/cloud-e2e.sh && cloudE2E'",
```

Comment:
why `bash -c` here?

Reply:
this change came from dev/upstream yarn2 migration, yarn2 requires this syntax to source & run the bash scripts now